pkg/commands/config.go

```diff
@@ -33,6 +44,16 @@ func newConfigCommand(log logutils.Log) *configCommand {
 		RunE: func(cmd *cobra.Command, _ []string) error {
 			return cmd.Help()
 		},
+		PersistentPreRunE: c.preRunE,
+	}
+
+	verifyCommand := &cobra.Command{
```

Comment:
Could we enhance the help message for the config verify command to display all possible flags?

Currently, it's not clear what should be provided:
```
❯ ./golangci-lint config verify -h
Verify configuration against JSON schema

Usage:
  golangci-lint config verify [flags]

Global Flags:
      --color string   Use color when printing; can be 'always', 'auto', or 'never' (default "auto")
  -c, --config PATH    Read config from file path PATH
  -h, --help           Help for a command
      --no-config      Don't read config file
  -v, --verbose        Verbose output
```

Yes, someone might look at this PR and notice the `--schema` parameter, but that's not a typical user.

Reply:
The help might look like this:

```
❯ ./golangci-lint config verify -h
Verify configuration against JSON schema

Usage:
  golangci-lint config verify [flags]

Flags:
  -c, --config PATH   Read config from file path PATH
      --no-config     Don't read config file
  -s, --schema        JSON schema path

Global Flags:
      --color string   Use color when printing; can be 'always', 'auto', or 'never' (default "auto")
  -h, --help           Help for a command
  -v, --verbose        Verbose output
```

Reply:
The flag is only for debugging purposes, it's not a user flag.

Reply:
It's already consistent:

````console
 $ ./golangci-lint config path -h                                            
Print used config path

Usage:
  golangci-lint config path [flags]

Global Flags:
      --color string   Use color when printing; can be 'always', 'auto', or 'never' (default "auto")
  -c, --config PATH    Read config from file path PATH
  -h, --help           Help for a command
      --no-config      Don't read config file
  -v, --verbose        Verbose output
```